README.md

```diff
@@ -179,13 +179,15 @@ $ kubectl delete -f example/example-etcd-cluster.yaml
 
 ### etcd operator recovery
 
-deLet's walk through in the following steps.
+Let's walk through operator recovery in the following steps.
+
+Create an etcd cluster:
 
 ```
 $ kubectl create -f example/example-etcd-cluster.yaml
 ```
 
-Wait until all three members are up. Then
+Wait until all three members are up. Then stop the etcd operator and delete one of the etcd clusters:
```

Comment:
... delete one of the etcd pods:

Reply:
@genome21 can you please make the wording change here and I'll merge after that.